scripts/send_log_config.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+from argparse import RawTextHelpFormatter
+import argparse
+import json
+import socket
+import struct
+
+
+parser = argparse.ArgumentParser(
+        description="""
+Send a python log config (json) to a port where a logging process listens.
+
+Futher deatils about the log configuration format and usage can be found here:
+https://docs.python.org/2/library/logging.config.html """,
+        formatter_class=RawTextHelpFormatter)
+
+parser.add_argument('-c', action="store", required="True",
+                    dest="config_file",
+                    help="Log configuration in json format.")
+
+parser.add_argument('-p', action="store", required="True",
```

Comment:
Maybe it would be great to create an argument for the `host` which would be equal with `localhost` by default.

Reply:
You can not send configuration to a remote server, only to `localhost`. I do not see why it should be configurable.